build.gradle

```diff
@@ -22,29 +22,33 @@ version = ["git", "describe", "--match", "[0-9]*", "--dirty"].execute().text.tri
 
 // Maps supported Android plugin versions to the versions of Gradle that support it
 def supportedVersions = [
-    "3.2.0-beta01": ["4.9"],
-    "3.1.4": ["4.4.1", "4.5.1", "4.6", "4.7", "4.8.1", "4.9"],
-    "3.1.3": ["4.4.1"],
-    "3.1.2": ["4.4.1"],
-    "3.1.1": ["4.4.1"],
-    "3.1.0": ["4.4.1"],
-    "3.0.1": ["4.1", "4.2.1", "4.3.1", "4.4.1", "4.5.1", "4.6", "4.7", "4.8.1", "4.9"],
-    "3.0.0": ["4.1"],
+    "4.0.0-beta04": ["6.1.1", "6.2.2", "6.3"],
+    "3.6.2": ["5.6.4", "6.0.1", "6.1.1", "6.2.2", "6.3"],
+    "3.6.1": ["5.6.4"],
+    "3.6.0": ["5.6.4"],
+    "3.5.3": ["5.4.1", "5.5.1", "5.6.4", "6.0.1", "6.1.1", "6.2.2", "6.3"],
+    "3.5.2": ["5.4.1"],
+    "3.5.1": ["5.4.1"],
+    "3.5.0": ["5.4.1"]
```

Comment:
It would be good to test with some 6.4 nightly, too.

Reply:
I've added a 6.4 nightly in here, too.